modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,27 @@
+[[hostname-rename]]
+= Rename {productname} Server
+
+It is possible to rename a {productname} Server.
+You first must change the network settings of the server locally and remotely.
+Then the [command]``spacewalk-hostname-rename`` script will the settings in the {productname} PostgreSQL database and the internal structures of {productname} accordingly.
+
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+
+
+
+.Procedure: Renaming {productname} Server
+. Change the network settings of the server on the system level locally and remotely at the DNS server.
+This part of the procedure is similar to the actions to take if you want to rename any other regular system.
++
+[NOTE]
+====
+You must also provide configuration settings for reverse name resolution.
+====
```

Comment:
I'm increasingly disliking admonitions in the middle of procedures, because our formatting (or lack thereof :confused: ) doesn't handle it well. Since it's only a note, can we just write it inline instead?

Reply:
ok